src/v/pandaproxy/schema_registry/avro.cc

```diff
@@ -262,8 +251,8 @@ result<void> sanitize(
                 }
             } else {
                 o.AddMember(
-                  rapidjson::Value("namespace", alloc),
-                  rapidjson::Value(
+                  json::Value("namespace"),
+                  json::Value(
                     new_namespace.data(), new_namespace.length(), alloc),
```

Comment:
should `alloc` be removed here like it is for the Value right above this?

Reply:
Good spot. This change was unintentional. The version that doesn't take the `alloc`, doesn't do a copy, it just assigns a pointer. In this case it's safe. I think I'll leave it.